commit eece932c3966e34ad3dfae41262ded87d58e09d3
Author: Magnus Lilja <lilja.magnus@gmail.com>
Date:   Wed Jan 25 22:07:59 2017 +0100

    usb: gadget: udc: fsl: Add missing complete function.
    
    commit 5528954a1a0c49c6974ef1b8d6eaceff536204d5 upstream.
    
    Commit 304f7e5e1d08 ("usb: gadget: Refactor request completion")
    removed check if req->req.complete is non-NULL, resulting in a NULL
    pointer derefence and a kernel panic.
    This patch adds an empty complete function instead of re-introducing
    the req->req.complete check.
    
    Fixes: 304f7e5e1d08 ("usb: gadget: Refactor request completion")
    
    Signed-off-by: Magnus Lilja <lilja.magnus@gmail.com>
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1312_eece932c3966e34ad3dfae41262ded87d58e09d3.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1312_eece932c3966e34ad3dfae41262ded87d58e09d3.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
